src/coreclr/System.Private.CoreLib/src/System/Diagnostics/SymbolStore/Token.cs

```diff
@@ -1,39 +1,23 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-/*============================================================
-**
-**
-** Small value class used by the SymbolStore package for passing
-** around metadata tokens.
-**
-===========================================================*/
-
 using System.Diagnostics.CodeAnalysis;
 
 namespace System.Diagnostics.SymbolStore
 {
-    internal struct SymbolToken
+    /// <summary>Small value class used by the SymbolStore package for passing around metadata tokens.</summary>
+    internal readonly struct SymbolToken : IEquatable<SymbolToken>
```

Comment:
It might be.  Looks like there's a copy of it in the System.Diagnostics.StackTrace assembly, so it's possible it's left over from before it was copied there.